plugins/woocommerce/tests/e2e-pw/tests/shopper/checkout-block.spec.js

```diff
@@ -167,7 +167,7 @@ test.describe(
 				enabled: true,
 			} );
 			// make sure our customer user has a pre-defined billing/shipping address
-			await api.put( `customers/2`, {
+			await api.put( `customers/${ process.env.CUSTOMER_USER_ID }`, {
```

Comment:
The main change in this PR: Replace the hardcoded customer ID with one that's obtained from querying the `/customers` endpoint in `global-setup`.